### PR TITLE
[fix] Add log in abort phase

### PR DESCRIPTION
**Pull request by DongLiang-0**
# Proposed changes

Issue Number: close #xxx

## Problem Summary:

Describe the overview of changes.
[fix] Add log in abort phase

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: flink-doris-connector/src/main/java/org/apache/doris/flink/sink/writer/DorisStreamLoad.java [hunk 1]
```diff
@@ -269,7 +271,7 @@ private void abortTransaction(long txnID) throws Exception {
         ObjectMapper mapper = new ObjectMapper();
         String loadResult = EntityUtils.toString(response.getEntity());
         Map<String, String> res = mapper.readValue(loadResult, new TypeReference<HashMap<String, String>>(){});
-        if (FAIL.equals(res.get("status"))) {
+        if (FAIL.equals(res.get("status")) || INTERNAL_ERROR.equals(res.get("status"))) {
```
Comment (JNSimba) [inline flink-doris-connector/src/main/java/org/apache/doris/flink/sink/writer/DorisStreamLoad.java:274]: Is it better to directly judge **not success**?

Comment (DongLiang-0) [reply flink-doris-connector/src/main/java/org/apache/doris/flink/sink/writer/DorisStreamLoad.java]: done

**Review by JNSimba — APPROVED** [review]
LGTM

**Review by JNSimba — APPROVED** [review]
LGTM

**Review by JNSimba — APPROVED** [review]
LGTM

**Pull request merged**